Objects/regions.c

```diff
@@ -643,9 +675,294 @@ PyObject* _Py_MakeImmutable(PyObject* obj)
 
     stack_free(frontier);
 
-    _Py_VPYDBGPRINT(obj);
-    _Py_VPYDBG(" region: %lu rc: %ld \n", Py_REGION(obj), Py_REFCNT(obj));
-    _Py_VPYDBG("<< makeimmutable complete\n\n");
 
-    return obj;
-}
\ No newline at end of file
+    Py_RETURN_NONE;
+}
+
+bool is_bridge_object(PyObject *op) {
+    Py_uintptr_t region = Py_GET_REGION(op);
+    if (is_default_region(region) || is_immutable_region(region)) {
+        return 0;
+    }
+
+    if ((Py_uintptr_t)((regionmetadata*)region)->bridge == (Py_uintptr_t)op) {
+        return 1;
+    } else {
+        return 0;
+    }
+}
+
+__attribute__((unused))
+static void regionmetadata_inc_lrc(regionmetadata* data) {
```

Comment:
I don't have a good feel yet when macros should be used and when functions. I would probably leave this refactoring/decision to @mjp41 when he implements the macros for the write barrier. It might be reasonable to use a macro there to have the update be inline instead of a function call.

Also to not block this PR on the design of these functions, unless you think we should